app/src/NativeBase/Components/Forms/ResponseModal.tsx

```diff
@@ -0,0 +1,89 @@
+/**
+ * @file Modal shows response when a form is submitted.
+ */
+
+import React, { FC } from 'react'
+import { Text, Modal, Icon } from 'native-base'
+import MaterialIcons from 'react-native-vector-icons/MaterialIcons'
+import StaTheme from '../../Theme/StaTheme'
+
+interface ResponseModalProps {
+  message: string
+  header: string
+  success: boolean
+  isOpen: boolean
+  onClose: (value: boolean) => void
+}
+
+const ResponseModal: FC<ResponseModalProps> = ({
+  message,
+  header,
+  success,
+  isOpen,
+  onClose,
+}) => {
+  if (success) {
+    return (
+      <Modal
+        isOpen={isOpen}
+        onClose={onClose}
+        justifyContent="center"
+        size="xl"
+      >
+        <Modal.Content
+          _dark={{ color: StaTheme.colors.bgDarkMode['100'] }}
+          _light={{ backgroundColor: StaTheme.colors.bg['100'] }}
+        >
+          <Modal.Header
```

Comment:
One other small suggestion: as a user my inclination was to try and tap on the modal to close it -- could allow the user to do this?  (It took me a few seconds to realise I needed to tap outside it)

Would just need a `<Pressable onPress={() => onClose(true)}>` above this line, wrapping around `Modal.Header` and `Modal.Body` (i.e. a closing tag `</Pressable>` after `</Modal.Body>`)